### DISCUSSION TITLE
ValueError

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#375] [Q&A] [answered]
Kenapa yg keluar ValueError waktu saya ngerun python app.py
![Capture](https://user-images.githubusercontent.com/112919414/193409455-bd1c3b5e-3db4-420f-9458-97737cd90410.JPG)

**Comment by ageljonathan** [comment]
Saya coba di komputer temen yg lain berhasil

**Reply by vincentvaldinata** [reply]
coba masuk ke environtmentnya dulu

**Reply by ageljonathan** [reply]
sudah masuk pak

**Comment by ageljonathan** [comment, marked as answer]
sudah berhasil
ternyata di versi matplotlib saya penulisan warna di barplot sebaiknya begini
my_colors = ["r", "g", "b", "w", "y", "m", "c"]
yg sblmnya begini
my_colors = 'rgbkymc'
